.github/workflows/build.yml

```diff
@@ -91,7 +91,7 @@ jobs:
               if: matrix.os == 'windows-2016'
 
             - name: Get Google Test
-              run: git clone --branch release-1.10.0 https://github.com/google/googletest.git external/googletest
```

Comment:
Same applies here - redundant if UPDATE_DEPS is used.

Reply:
Removed.